Hatten_CletusPA1.asm

```diff
@@ -0,0 +1,90 @@
+.data
+	prompt: 	.asciiz "How many numbers do you want to input? "
+	message: 	.asciiz "\nThe number of inputs you have chosen is "
+	spaceHere:	.asciiz "\n"
+	
+	fibs:   	.word 0:9
+	sizelimit: 	.word 9
+
+.text
+	li $v0, 4		#Prompt user for input
+	la $a0, prompt
+	syscall 
+	li $v0, 5		#Stores user's input
+	syscall
+	move $t0, $v0		#Stores user's input inside temporary file
+	li $v0, 4		#Displays message
+	la $a0, message
+	syscall
+	li $v0, 1		#Displays user's input
+	move $a0, $t0		#Stores $t0 into an argument
+	syscall
+	move $s0, $a0		#Stores argument into a saved value
+	li $v0, 4		#Creates space in between user's input and next prompt
+	la $a0, spaceHere
+	syscall
+	la   $s1, fibs        	# load address of array into $s0
+	la   $s2, sizelimit
+	move $s6, $s2       	# load address of size variable into $s6
```

Comment:
-2 Prints nothing for input greater than 9.  Would be more appropriate to use a branch to prompt for input again or to simply force printing of 9 numbers only.